dependencies.yml

```diff
@@ -25,8 +25,14 @@ com.squareup.retrofit2:
   adapter-java8: { version: *RETROFIT2_VERSION }
   converter-jackson: { version: *RETROFIT2_VERSION }
 
+com.ryantenney.metrics:
+    metrics-spring: { version: 3.1.3 }
```

Comment:
- This should come before `com.spotify` because R comes before S.
- Indentation (4 -> 2 spaces)